packages/react-widgets/src/DropdownList.js

```diff
@@ -493,8 +493,7 @@ class DropdownList extends React.Component {
       'search',
       () => {
         var list = this.list,
-          key = this.props.open ? 'focusedItem' : 'selectedItem',
-          item = list.next(this.state[key], word)
+          item = list.next(null, word)
```

Comment:
this isn't quite right. I think it should start from the current item and loop back around

Reply:
Do you have any concerns about starting the search from the beginning?

Reply:
i think it feels less intuitive, maybe. I'm not sure, if it's reasonably easy i think i'd prefer it. At the very least it would bring back the behavior it used to have